client/assets/content_bundles/symptom_checker.en.yaml

```diff
@@ -0,0 +1,379 @@
+schema_version: 1
```

Comment:
@brunobowden is this the content created entirely by WHO/team members, or is this the wording made earlier on....?

Reply:
@advayDev1 - this is the same content as was removed in #1476. I've asked @patniemeyer to restore it as it was so we can check that it all works. There will be a separate process with WHO / Nigeria to figure out what the policy and text will be, likely that'll be something different from what it currently is.

Reply:
See slack.  This was removed for a reason.

Reply:
I'm not sure what to do about this so I'm going to leave it in the PR for now.